automation/tables.py

```diff
@@ -1,18 +1,15 @@
-import enum
-
 from . import models
 from .functions import inbound, members
-from .utils import airtable
+from .clients import airtable
 
+INBOUND = airtable.TableSpec(
+    name="inbound",
+    model_cls=models.InboundModel,
+    status_to_cb={None: inbound.NewCallback},
```

Comment:
This is probably the oddest part right now, where `status_to_cb` is now a map from status to a type that expects to be constructed with a `Config` instance (in case it needs to construct clients for other services).

This (`status_to_cb`, and how `poll_table` uses it) is the thing I'd like to clean up next, so please ignore this weirdness for the moment.